rbac/management/audit_log/view.py

```diff
@@ -33,7 +33,13 @@ class AuditLogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
     serializer_class = AuditLogSerializer
     permission_classes = (AuditLogAccessPermission,)
 
+    def filter_by_id(self, queryset):
+        """Filter queryset by id."""
+        return queryset.order_by("-id").values()
+
     def list(self, request, *args, **kwargs):
         """List all of the audit logs within database by tenant."""
         self.queryset = filter_queryset_by_tenant(AuditLog.objects.all(), request.tenant)
+        if request.query_params.get("order_by") is not None:
+            self.queryset = self.filter_by_id(self.queryset)
```

Comment:
I was not able to find any open api documentation to that view, it is in progress still? it is related with my previous comment, what is the expected usage of this `order_by` query param? because now the value is not validated, this only checks if that param is included or not, but maybe I dont see the context 😇

Reply:
Oh! I completley meant to be ordering by id! I wanted to have it ordered by ID so when I was testing the audit logs, I was able to see the most recent audit log appear on top so that I didn't need to go looking for it!! 
I'll update the method name and docstring!! 😅

Reply:
thank you Ellen, is there some plan to add these into open api spec as well? the endpoint `/api/rbac/v1/auditlogs/` is working right and can be used?

Reply:
Yes, the endpoint is working!! 
I do plan to add them into the open api spec!! Thank you for reminding me about it!! I'll update it within the next PR!!

Reply:
awesome, thank you